### DISCUSSION TITLE
"Git diff" view between two prompt versions

### DISCUSSION THREAD

**Discussion by mukundt** [langfuse/langfuse#1105] [Ideas]
### Describe the feature or potential improvement

Having a "git diff"-style view between prompt versions would help us understand changes.

### Contribute

- [ ] Yes, I can implement this and raise a PR

**Comment by marcklingen** [comment]
Thanks for the suggestion, agree, this'd be neat.

Had a quick look and there are some good diff renderers for React like this one: https://github.com/praneshr/react-diff-viewer

**Comment by hassiebp** [comment]
We have just shipped the side-by-side diff view across prompt versions and prompt creation: 

![CleanShot 2025-01-22 at 18 22 55](https://github.com/user-attachments/assets/9dec41e7-a1ac-4ef7-afbd-321f299449fb)
